libs/core/src/components/pds-popover/pds-popover.scss

```diff
@@ -0,0 +1,169 @@
+:host {
+  --border-radius-overlay: var(--pine-border-radius-125);
+
+  --border-width-arrow-down: var(--sizing-arrow-default) var(--sizing-arrow-default) 0;
+  --border-width-arrow-left: var(--sizing-arrow-default) var(--sizing-arrow-default) var(--sizing-arrow-default) 0;
+  --border-width-arrow-right: var(--sizing-arrow-default) 0 var(--sizing-arrow-default) var(--sizing-arrow-default);
+  --border-width-arrow-up: 0 var(--sizing-arrow-default) var(--sizing-arrow-default);
+
+  --box-shadow-default: var(--pine-box-shadow-200);
+
+  --color-background-default: var(--pine-color-white);
+  --color-text-default: var(--pine-color-grey-950);
+
+  --sizing-arrow-default: 6px;
+  --sizing-arrow-offset: 14px;
+  --sizing-width-default: 320px;
+
+  --spacing-padding-overlay: var(--pine-spacing-100) calc(var(--pine-spacing-300) / 2);
+
+  display: inline-block;
+  position: relative;
+
+  ::slotted(*) {
+    display: flex;
+  }
+
+  ::slotted([slot="content"]) {
+    display: block;
+    white-space: normal;
+    width: var(--sizing-width-default);
+  }
+}
+
+.pds-popover__content {
+  background-color: var(--color-background-default);
+  border-radius: var(--border-radius-overlay);
+  box-shadow: var(--box-shadow-default);
+  color: var(--color-text-default);
+  // TODO: need to use block / none but the popover content width and height are needed for calculations
+  opacity: 0;
+  padding: var(--spacing-padding-overlay);
+  position: absolute;
+  visibility: hidden;
+  width: max-content;
```

Comment:
Do you think we need some type of max-width or something here? For the tooltip, we made it somewhat customizable due to long strings causing some very wide tooltips. I didn't fully test, but just more of a heads up.

Reply:
+1